client/sites-dashboard/components/sites-table-filter-tabs.tsx

```diff
@@ -8,25 +10,70 @@ interface SitesTableFilterTabsProps {
 	className?: string;
 }
 
+interface FilteredSites {
+	[ name: string ]: SiteData[];
+}
+
+interface SiteTab extends Omit< TabPanel.Tab, 'title' > {
+	title: React.ReactChild;
+}
+
+const SitesTabPanel = styled( TabPanel )`
+	.components-tab-panel__tabs {
+		overflow-x: auto;
+		padding-bottom: 10px;
+	}
+
+	.components-tab-panel__tabs-item {
+		--wp-admin-theme-color: var( --studio-gray-100 );
+		color: var( --studio-gray-60 );
+		padding: 0;
+		margin-right: 24px;
+		font-size: 16px;
+		flex-shrink: 0;
+	}
+	.components-tab-panel__tabs-item:hover,
+	.components-tab-panel__tabs-item.is-active,
+	.components-tab-panel__tabs-item.is-active:focus,
+	.components-tab-panel__tabs-item:focus:not( :disabled ) {
+		box-shadow: inset 0 -2px 0 0 var( --wp-admin-theme-color );
+		color: var( --studio-gray-100 );
+	}
+`;
+
 export function SitesTableFilterTabs( {
 	allSites,
 	children: renderContents,
 	className,
 }: SitesTableFilterTabsProps ) {
 	const { __ } = useI18n();
 
+	let tabs: SiteTab[] = [
+		{ name: 'all', title: __( 'All' ) },
+		{ name: 'launched', title: __( 'Launched' ) },
+		{ name: 'private', title: __( 'Private' ) },
+		{ name: 'coming-soon', title: __( 'Coming soon' ) },
```

Comment:
@p-jackson should we include the namespace in those `__` calls there? The previous version did not have it. I'm trying to understand when and when not to use it!

Reply:
The `SitesTableFilterTabs` code had to move out of the packages folder and into the Calypso app code to get the unit tests to pass. It's only packages that need the `__i18n_text_domain__` constant to be used.

Hypothetically it'd be ok for the `__i18n_text_domain__` constant to be in Calypso app code too, but I'm guessing @sejas removed it because the tooling hasn't been set up for this project and so there'd be errors.

Eventually when the issue that caused us to move `SitesTableFilterTabs` out of the packages folder is fixed upstream then we'll need to put the text domain constant back in.